### DISCUSSION TITLE
Daily detailed e-mail reports?

### DISCUSSION THREAD

**Discussion by antdude** [fail2ban/fail2ban#3272] [Q&A] [answered]
Hello.

I used to use DenyHosts in older Debian versions like v8 jessie. Since I just did a brand new clean installation of Debian bullseye v11.3, but it no longer carries useful DenyHosts to block annoying SSH brute attacks on default port 22 (can't use another number due to some places blocking non-default numbers). :(

So, I am trying out fail2ban v0.11.2 I think I have it set up and working (see bans and unbans in /var/log/fail2ban.log). How can I get e-mail notifications of a daily detailed summary report of the attacks like what login names, passwords, addresses, etc.? This will be on localhost (e.g., root to ant) using exim4.

Thank you for reading and hopefully answering soon. :)

**Comment by sebres** [comment, marked as answer]
> How can I get e-mail notifications of a daily detailed summary report of the attacks like what login names, passwords, addresses, etc.?

Unfortunately, there is no such feature in fail2ban at the moment. 
But you can write some cron sending output of `fail2ban-client status` per mail.
